src/components/utils.js

```diff
@@ -8,3 +8,6 @@ export const isValidAddress = address => {
     return false;
   }
 };
+
+export const isValidHash = input => /^0x[a-fA-F0-9]{64}$/.test(input);
+export const isEmpty = input => input.length === 0;
```

Comment:
Most case it makes sense to just use `!value`

Reply:
is input an array ? or collection ? maybe can dirctly use lodash.isEmpty

Reply:
maybe need to add sme jsdoc abut the expected input type